lightblue-ldap-config/src/main/java/com/redhat/lightblue/config/ldap/LdapDBResolver.java

```diff
@@ -68,4 +70,20 @@ private LdapDataSourceConfiguration findByDatabase(String database){
         return null;
     }
 
+    @Override
+    public Map<String, Object> getLDAPConnectionsStatus() {
+        
+        Map<String, Object> connectionsStatus = new HashMap<>();
+        LDAPConnection connection = null;
+        for (LdapDataSourceConfiguration ldapDS : ldapDataSources) {
+            try {
+                connection = ldapDS.getLdapConnection();
+                connectionsStatus.put(ldapDS.getDatabaseName(), connection.isConnected());
+            } catch (LDAPException e) {
```

Comment:
I would suggest, same as lightblue-rest, of using one of the connection pool's health check methods for richer health check behavior.